forge-gui/res/cardsfolder/upcoming/djinn_of_the_fountain.txt

```diff
@@ -0,0 +1,15 @@
+Name:Djinn of the Fountain
+ManaCost:4 U U
+Types:Creature Djinn
+PT:4/4
+K:Flying
+T:Mode$ SpellCast | TriggerZones$ Battlefield | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | NoResolvingCheck$ True | Execute$ TrigCharm | TriggerDescription$ Whenever you cast an instant or sorcery spell, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBPump,DBExile,DBScry
+SVar:DBPump:DB$ Pump | Defined$ Self | NumAtt$ 1 | NumDef$ 1 | SpellDescription$ CARDNAME gets +1/+1 until end of turn.
+SVar:DBExile:DB$ ChangeZone | Defined$ Self | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | RememberChanged$ True | SpellDescription$ Exile CARDNAME. Return it to the battlefield under its owner's control at the beginning of the next end step.
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | ConditionDefined$ Remembered | ConditionPresent$ Card | TriggerDescription$ Return CARDNAME to the battlefield.
```

Comment:
I think make the TrigDesc closer to the Oracle... maybe:
`Return CARDNAME to the battlefield under its owner's control at the beginning of the next end step.`